code/modules/persistence/noticeboards.dm

```diff
@@ -7,7 +7,7 @@
 	anchored = 1
 	var/list/notices
 	var/base_icon_state = "nboard0"
-	var/const/max_notices = 5
+	var/const/max_notices = 35
```

Comment:
> /obj/structure/noticeboard/update_icon()
> 	icon_state = "[base_icon_state][LAZYLEN(notices)]"		icon_state = "[base_icon_state][LAZYLEN(notices)]"

Take note of this code, only reason why I stalled doing this. You might want to find out a way for the icon not to disappear when it exceeds 5.

Reply:
Fixed